buildSrc/src/main/java/Dependencies.kt

```diff
@@ -35,7 +35,7 @@ object Versions {
     const val androidx_work = "2.0.1"
     const val google_material = "1.1.0-alpha10"
 
-    const val mozilla_android_components = "14.0.1"
+    const val mozilla_android_components = "15.0.0-SNAPSHOT"
```

Comment:
Not sure if we want the upgrade as part of this commit. As the new categories are already exposed on `14.0.1`